src/ui/screens/home/controller/ControllerView.qml

```diff
@@ -686,12 +648,11 @@ Item {
         accessibleName: ipInfoPanel.isOpen
             ? connectionInfoCloseText
             : MZI18n.ConnectionInfoConnectionInformation
+        Accessible.ignored: !enabled || !visible
         buttonColorScheme: MZTheme.theme.iconButtonDarkBackground
         enabled: visible && VPNConnectionHealth.stability !== VPNConnectionHealth.NoSignal
         opacity: visible ? 1 : 0
-        visible: connectionInfoToggleButton.visible
-            && !connectionInfoScreen.isOpen
-            && !connectionInfoScreen.isTransitioning
+        visible: !connectionInfoScreen.isOpen && !connectionInfoScreen.isTransitioning
```

Comment:
Won't this make it IpInfo button visible when the VPN is off?

Reply:
Yes. Thanks for catching.

Reply:
Fixed.